depends/funcs.mk

```diff
@@ -39,8 +39,20 @@ define fetch_file
       $(call fetch_file_inner,$(1),$(FALLBACK_DOWNLOAD_PATH),$(3),$(4),$(5))))
 endef
 
+define int_get_local_dir
+$(eval $(1)_sha256_hash:=$(shell
+    if ! [ -f $($(1)_fetched) ] || [ $($(1)_fetched) -ot $($(1)_local_dir) ]; then \
+        mkdir -p $(dir $($(1)_fetched)); \
+        $(build_TAR) -c --sort=name --mtime=0 --owner=0 --group=0 --numeric-owner -f $($(1)_source) -C $($(1)_local_dir) . && \
```

Comment:
01452f8c0da58db549bc46d0cfa7de715344efc4: the Apple version of tar doesn't have `--sort`, see https://github.com/chaincodelabs/libmultiprocess/issues/139#issuecomment-2621080394